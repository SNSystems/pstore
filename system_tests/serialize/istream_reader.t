# %S = the test source directory
# %binaries = the directories containing the executable binaries
# %t = temporary file name unique to the test

REQUIRES: examples
RUN: "%binaries/example-istream-reader" > "%t"
RUN: diff -b "%t" "%S/istream_reader_expected.txt"
