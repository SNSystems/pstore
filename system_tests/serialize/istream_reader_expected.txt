3
73
127
179
